source/developers/license.markdown

```diff
@@ -0,0 +1,37 @@
+---
+layout: page
+title: "The MIT License (MIT)"
+description: "The License that all Home Assistant projects are licensed under"
+date: 2017-01-21 15:05:00 -0800
+sidebar: true
+comments: false
+sharing: true
+footer: true
+---
+
+Copyright © `2017` `Paulus Schoutsen`
+
+Permission is hereby granted, free of charge, to any person
+obtaining a copy of this software and associated documentation
+files (the “Software”), to deal in the Software without
+restriction, including without limitation the rights to use,
+copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the
+Software is furnished to do so, subject to the following
+conditions:
+
+The above copyright notice and this permission notice shall be
+included in all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND,
+EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES
+OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,
+WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+OTHER DEALINGS IN THE SOFTWARE.
+
+## Adoption
```

Comment:
I'm unsure about this one. It makes sense when looking at the raw text but when looking at this parsed with Markdown (like it will 99% of the time) it is clearly a separate section.